### DISCUSSION TITLE
Color Theme

### DISCUSSION THREAD

**Discussion by KristofPusztai** [marimo-team/marimo#1073] [Ideas]
In streamlit there is the ability to change [color theme](https://docs.streamlit.io/develop/concepts/configuration/theming) of the rendered web application, could be a useful addition here as well! I know that you can edit the style using the `.style` method that gets inherited from the `Html` object, however, for more complex elements such as UI Sliders, it does not appear to be trivial to change the colors of these. 

Adding this as an option to the *.marimo.toml* file, similar to the streamlit implementation, would be extremely helpful imo!

**Comment by KristofPusztai** [comment]
Browsing the issues, looks like someone else [also had this thought ](https://github.com/marimo-team/marimo/issues/989)but put it as an issue instead!

**Reply by akshayka** [reply]
Thanks for the nudge; we will get to this eventually!
